ci/Dockerfile

```diff
@@ -31,5 +31,6 @@ COPY ci/requirements.txt requirements.txt
 RUN pip install --no-cache-dir -r requirements.txt ${EXTRA_INDEX}
 
 WORKDIR /oml-lib
+ENV PYTHONPATH=$PYTHONPATH:/oml-lib
```

Comment:
@AlekseySh 
https://github.com/OML-Team/open-metric-learning/actions/runs/3996595545/jobs/6974148053
I suspect that tests fail because of this line.

I had to set it in order to be able to run examples tests. May be I should set it in makefile before tests.

Without it tests failed on import of `examples` module.

Reply:
There was an issue with pip the last weekend (they changed something without back-compatibility). I've already made some fixes on our side, so, you need to merge the main to your branch. Or you can perform this changes manually, see the recent commit to the main branch named "fix ci"

Reply:
@churnikov ^

Reply:
Sorry for the delay.

I got sick and was not able to work :(

Right now every test passes as far as I can see.